### DISCUSSION TITLE
how to train a 70s tv?

### DISCUSSION THREAD

**Discussion by svumme** [TheLastBen/fast-stable-diffusion#1215] [General]
I have 18 photos of a specific TV from the 70s, I took them in the studio with a gray photographic backdrop, I tried to use these parameters: unet 3000 step and the Text_Encoder_Training_Steps 250 but I didn't get good results, they were a bit the parameters I had used for people. I don't know if it is necessary to put concept images and if so what kind of images should be used? Could you recommend some settings to start with? I would be very grateful.happy new year!

**Comment by TheLastBen** [comment]
for a 2e-5 learning rate, all you need is 600 unet steps
